index.html

```diff
@@ -255,5 +255,46 @@ <h3>LET’S CONNECT</h3>
         </div>
       </div>
     </section>
+    <footer>
+      <div class="contact-form"></div>
+      <h2 class="contact-me">Contact me</h2>
+      <p class="para-text">
+        If you have an application you are interested in developing, a feature
+        that you need built or a project that needs coding. I’d love to help
+        with it
+      </p>
+      <form action="#" id="comment">
+        <input
+          type="text"
+          id="name"
+          placeholder="Kumilachew"
+          minlength="1"
+          maxlength="30"
+          required
+        />
+        <input
+          type="email"
+          id="email"
+          placeholder="kumilachew1212@gmail.com"
+          required
+        />
+        <textarea
+          form="comment"
+          placeholder="Write your message here"
+          required
+          minlength="1"
+          maxlength="500"
+          name="msg"
+          id="msg"
+          cols="30"
+          rows="10"
+        ></textarea>
+        <button type="submit" class="btn">Get In Touch</button>
+      </form>
+
+      <div class="ending">
+        <img src="images/Shape@2x.svg" alt="shape" />
+      </div>
+    </footer>
```

Comment:
- [ ] Your contact form is scrolling horizontally kindly fix it
### Your design
![hiss](https://user-images.githubusercontent.com/81727988/177508666-824843eb-82c5-48f5-b6ce-c0956033abef.png)